packages/frontend/src/components/Swap.tsx

```diff
@@ -96,23 +106,33 @@ export default function Trade({ handleOpenModal }: Props) {
         });
         setFetchingPrice(false);
       } catch (error: any) {
-        setErrorMsg('Something Went wrong, please try again!');
+        setalertMsg('Something Went wrong, please try again!');
         setFetchingPrice(false);
       }
     }
   }
 
-  if (CGTcontractBalance == '' && CGTBalance == '') {
+  if (CGTcontractBalance === '' && CGTBalance === '') {
```

Comment:
change to
if (!CGTcontractBalance && !CGTBalance)
Tip: !'', !undefined, !null, !0 ... = true